### PR TITLE
change abstractor to use props

### PR DESCRIPTION
**Pull request by mgramigna**
functionality is the same

Posting this before I leave. Might need some tweaks but in a testable state

### CODE REVIEW

**Review by pmadria — APPROVED** [review]
I went over the changes and everything looks good and works as intended

File: src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
+  const { patientData } = usePatient();
+
+  useEffect(() => {
+    async function load() {
+      // Load Questionnaire
+      const questionnaireLoader = new QuestionnaireLoader();
+      const url = './static/mcode-questionnaire.json';
+      try {
+        const questionnaireResource = await questionnaireLoader.getFromUrl(url);
+        setQuestionnaire(questionnaireResource);
```
Comment (mickohanlon23) [inline src/App.tsx:24]: I'm getting a Typing error for `questionnaireResource`, but the app still seems to be running as expected. Is this okay for now?

File: src/App.tsx [hunk 2]
```diff
@@ -1,13 +1,61 @@
...
+          const fhirLibrary = response.data as R4.ILibrary;
+          const library = await new LibraryLoader(fhirLibrary).fetchELM();
+
+          const vsResponse = await axios.get('./static/mcode-valuesets.json');
+          const valueSetBundle = vsResponse.data as R4.IBundle;
+          const valueSetLoader = new ValueSetLoader(fhirLibrary, valueSetBundle);
+          const valueSetMap = valueSetLoader.seedValueSets();
+
+          setLibrary(library);
+          setValueSetMap(valueSetMap);
```
Comment (mickohanlon23) [inline src/App.tsx:42]: Same Typing error as above for `valueSetMap`

File: src/components/Abstractor/__tests__/Abstractor.test.tsx [hunk 3]
```diff
@@ -3,5 +3,11 @@ import { render } from '@testing-library/react';
 import Abstractor from '../Abstractor';
 
 test('renders Abstractor', () => {
+  global.LForms = {
```
Comment (mickohanlon23) [inline src/components/Abstractor/__tests__/Abstractor.test.tsx:6]: Also seeing some type errors here, but not sure about how significant this is. I know some of these already existed in master.

Comment (mickohanlon23) [reply src/components/Abstractor/__tests__/Abstractor.test.tsx:6]: The test is passing and its compiling so I think it should be fine.

**Review by mickohanlon23 — COMMENTED** [review]
Functionality still seems to work the same. Code is compiling and the questionnaire form renders properly. Besides those typescript errors (which I'm not sure of how important they are) everything looks solid!

**Comment by mgramigna** [comment]
That's funny. The typescript error's weren't showing up in Vim but were in VSCode. Either way, I think I resolved them.

**Review by mickohanlon23 — COMMENTED** [review]
I think this already existed in Master, but there is still a typing error within PatientProvider.tsx. Is this a quick fix now that we are using props?
Screen Shot 2020-09-14 at 4 32 45 PM

**Review by mickohanlon23 — APPROVED** [review]
PatientProvider type is happy now. Results are back now that you're awaiting the `seedValueSets()` call. Everything else still operates as expected. Looks great!

**Pull request merged**